DiffPlex/Chunkers/CharacterChunker.cs

```diff
@@ -0,0 +1,12 @@
+﻿namespace DiffPlex.Chunkers
+{
+    public class CharacterChunker:IChunker
+    {
+        public string[] Chunk(string text)
+        {
+            var s = new string[text.Length];
+            for (int i = 0; i < text.Length; i++) s[i] = text[i].ToString();
```

Comment:
I don't have a linter setup, but I generally like to always have curly brackets for consistency

Reply:
This code was extracted from the existing code base as it is. If you insist I can reformat it. By the way, I also prefer wrapping code blocks in the curly braces.

Reply:
Clearly, a much younger version of me did not ;) 
No need to reformat.